From 1d862091e7b7f49eb8533c74b6f7de34d6feb56c Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Mon, 12 Aug 2019 15:58:07 +0300
Subject: [PATCH] sdk_dpaa: ceetm: update Makefile to use absolute include
 paths

Explicitly mention the root source tree directory when building. This
allows the build process to start from a different location.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/freescale/sdk_dpaa/Makefile
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/Makefile
@@ -18,7 +18,7 @@ ifeq ($(CONFIG_FSL_DPAA_1588),y)
 fsl_dpa-objs += dpaa_1588.o
 endif
 ifeq ($(CONFIG_FSL_DPAA_CEETM),y)
-ccflags-y += -Idrivers/net/ethernet/freescale/sdk_fman/src/wrapper
+ccflags-y += -I$(srctree)/drivers/net/ethernet/freescale/sdk_fman/src/wrapper
 fsl_dpa-objs += dpaa_eth_ceetm.o
 endif
 
